docs/src/main/asciidoc/resource-handling.adoc

```diff
@@ -158,27 +163,31 @@ Resource[] fileTextResources = azureStorageFileProtocolResolver.getResources("az
 
 ==== Handling with resource 
 
-===== Download data from specific resource.
+===== Download data from specific resource
 
 You can download a resource from Azure Blob or file storage with the `getInputStream()` method of Resource.
 
+[source,java]
 ----
 @Value("azure-blob://[your-container-name]/[your-blob-name]") 
 private Resource storageBlobResource;
 
 @Value("azure-file://[your-fileshare-name]/[your-file-name]") 
 private Resource storageFileResource;
+
 ....
+
 // download data as stream from blob resource
 InputStream inputblobStream = storageBlobResource.getInputStream();
 // download data as stream from file resource
 InputStream inputfileStream = storageFileResource.getInputStream();
 ----
 
-===== Upload data to specific resource.
+===== Upload data to specific resource
 
 You can upload to a resource to Azure Blob or file storage by casting the Spring `Resource` to `WritableResource`.
```

Comment:
--> You can upload a resource to Azure Storage Blob or File Share by casting the Spring `Resource` to `WritableResource`.